src/foam/box/SessionServerBox.java

```diff
@@ -53,6 +54,7 @@ public void send(Message msg) {
         session = new Session();
         session.setId(sessionID == null ? "anonymous" : sessionID);
         session.setRemoteHost(req.getRemoteHost());
+        session.setCreated(new Date());
```

Comment:
Why is this necessary? Here is the definition of the created property:
    {
      class: 'DateTime',
      name: 'created',
      factory: function() { return new Date(); },
      javaFactory: 'return new Date();'
    },

which should do this already. Do we have a bug in FOAM?

Reply:
Factories are not triggered at object creation time, but at first access, so this will return the time that the created property is first accessed.